content/en/blog/general/introducing-reports-server/index.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2024-04-24
+title: Kyverno Reports Server - The ultimate solution to scale reporting
+linkTitle: Kyverno Reports Server - The ultimate solution to scale reporting
+author: Vishal Choudhary
+description: Security or Scalability? Why not both!
+---
+
+## Introduction
+
+Policy Reports are used by Kyverno to store the result of policies and cluster policies that match a resource. Kyverno generates reports during admission request as well as periodically as background scans. They are very helpful in auditing the current state of policy compliance in a cluster.
+
+Kyverno also creates its own custom ephemeral reports which are later aggregated to create the final policy reports or cluster policy reports. Policy reports and ephemeral reports are stored in etcd as Custom Kubernetes Resources. 
+
+This setup works fine in most cases but we start hitting the limits Kubernetes API server in a large cluster. During heavy reporting, the volume of data being written and retrieved by etcd puts the API server under severe load that can lead to poor performance. Moreover, etcd has a maximium capacity limit therefore, it can have a limited number of resources in it. This limit can be reached in large cluster with a lot of report producers.
```

Comment:
```suggestion
This setup works fine in most cases, but in larger-scale environments the limits of the Kubernetes API server could be reached. During periods of especially heavy reporting, the volume of data being written to and read from etcd can put the API server under severe load which can lead to degraded performance. Additionally, etcd has a maximum capacity limit and therefore has a limited number of resources it may store. This limit can be reached in large clusters with many report producers.
```